### ISSUE TITLE
All/none layer selector checkbox

### ISSUE THREAD

**Issue by ahayes** [GCRC/nunaliit#41]
As the number of layers increases, the ability to select and/or deselect all layers could be useful, and could also be an expedient way to bring a browser to its knees. Is there a balance here?

**Comment by boikle** [comment]
Now that the layer selector is going to be replaced with the new legend widget. Did we want this functionality in the new legend widget selector?

**Comment by ahayes** [comment]
Yes. We need to try out the new widgetlegend2 code ( #859 ) and see if it has the ability to toggle groups and/or all.
